apps/api/main.py

```
@@ -53,6 +58,8 @@ async def lifespan(app: FastAPI):
 # Add Langfuse Middleware (uncomment when configured)
 # app.add_middleware(LangfuseMiddleware)
 
+import logging # Add this import for the getLogger calls in lifespan
```

Ruff (0.11.9)

Module level import not at top of file

(E402)